editor/src/core/model/element-metadata-utils.ts

```diff
@@ -1516,16 +1516,21 @@ export const MetadataUtils = {
     metadata: ElementInstanceMetadataMap,
     path: ElementPath,
   ): JSXElement | null {
-    const element = MetadataUtils.findElementByElementPath(metadata, path)
+    return this.getJSXElementFromElementInstanceMetadata(
```

Comment:
I know we have used it in a few places in this file, but please do avoid using `this` here
```suggestion
    return MetadataUtils.getJSXElementFromElementInstanceMetadata(
```